zadania_ruby/first_break/sum_array.rb

```diff
@@ -0,0 +1,15 @@
+def sum_array(array)
+  new_array = array.reject { |elem| elem == array.max || elem == array.min }
+  index = 0
+  sum = 0
+  while index < new_array.size
```

Comment:
A jakby to wyglądało jakbyś skorzystała z https://apidock.com/rails/Enumerable/each_with_object ?